### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by ps-jarvis**

### CODE REVIEW

**Comment by Quetzacoalt91** [comment]
@Hlavtox I had some issues in the French file where modification where done by the PrestaShop account. The old translations were applied (I don't know if this was a manual or automatic action).

Can you check the Czech one pleqse?

**Comment by Hlavtox** [comment]
@Quetzacoalt91 Hey Thomas, yes there are some issues, it seems that it automatically changed some of manual translations I made to the automatic ones.

Basically, all the wordings on the left are those I manually fixed.

**Comment by Quetzacoalt91** [comment]
All right, I'll have a look to restore your version.

**Comment by Hlavtox** [comment]
@Quetzacoalt91 It's few clicks only, finding the cause is more important ;-)

**Comment by Quetzacoalt91** [comment]
Obviously :)

File: translations/ModulesAutoupgradeAdmin.fr.xlf [hunk 1]
```diff
@@ -2205,7 +2205,7 @@ Comment: check DB access</note>
       </trans-unit>
       <trans-unit id="71aa2b9774b005c47f9cd225766b2045" approved="yes">
         <source>PrestaShop Original version</source>
-        <target state="final">Version non modifiée de PrestaShop</target>
+        <target state="final">version non modifiée de PrestaShop</target>
```
Comment (Quetzacoalt91) [inline translations/ModulesAutoupgradeAdmin.fr.xlf:2208]: Bad translation

File: translations/ModulesAutoupgradeAdmin.fr.xlf [hunk 2]
```diff
@@ -2126,7 +2126,7 @@ Comment: check DB access</note>
       </trans-unit>
       <trans-unit id="b77d20ed1ba7b2ee7139dc4f3fd13117" approved="yes">
         <source>Allow major upgrade:</source>
-        <target state="final">Autoriser les mises à jour majeures :</target>
+        <target state="final">Autoriser les mise à jour majeure :</target>
```
Comment (Quetzacoalt91) [inline translations/ModulesAutoupgradeAdmin.fr.xlf:2129]: Bad translation

File: translations/ModulesAutoupgradeAdmin.fr.xlf [hunk 3]
```diff
@@ -1864,7 +1864,7 @@ Comment: check DB access</note>
       </trans-unit>
       <trans-unit id="e3ea2f6fe56c80761e37fd9c393d88a5" approved="yes">
         <source><![CDATA[Enable maintenance mode and add your maintenance IP in [1]Shop parameters > General > Maintenance[/1]]]></source>
-        <target state="final"><![CDATA[Activez le mode de maintenance et ajoutez votre IP de maintenance dans [1]Paramètres de la boutique > Paramètres généraux > Maintenance[/1]]]></target>
+        <target state="final"><![CDATA[Activez le mode de maintenance et ajoutez votre IP de maintenance dans [1]Paramètres de la boutique > Paramètres généraux > Maintenance[/1].]]></target>
```
Comment (Quetzacoalt91) [inline translations/ModulesAutoupgradeAdmin.fr.xlf:1867]: Bad translation

**Review by Quetzacoalt91 — CHANGES_REQUESTED** [review]

**Review by Quetzacoalt91 — APPROVED** [review]

**Pull request merged**